CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+# 1Password Community Code of Conduct
+The Passage by 1Password GitHub exists to facilitate information sharing and deliver resources to developers who are interested in Passage products.
+
+We want to encourage an environment that is friendly, safe, accepting, and free from intimidation and harassment.
+
+To this end, certain behaviors and practices will not be tolerated.
+
+## TL;DR
+- Be respectful
+- No solicitation
+- Abusive behavior is never tolerated
+
+Our community administrators are here to help:
+- [Sara Teare (Co-founder)](https://1password-devs.slack.com/team/U032CAY0DAB)
```

Comment:
So we share these contacts with 1Password?

Reply:
That's my assumption yes, for all the like "help/support" stuff I just point them to the 1Password developer slack with hyperlinks

Reply:
I'm not sure this is a valid assumption. We wouldn't want them to reach out to Sara or any 1Password folks until they have the necessary Passage context. We should send them to support@passage.id for now.

Reply:
I'll make that change